test/integration/other/migrations.js

```diff
@@ -33,6 +32,23 @@ const up = () => withTestDatabase((migrator) =>
 const down = () => withTestDatabase((migrator) =>
   migrator.migrate.down({ directory: migrationsDir }));
 
+const testMigration = (filename, tests, options = {}) => {
+  const { only = false, skip = false } = options;
+  const f = only ? describe.only : (skip ? describe.skip : describe);
```

Comment:
Bind `.only` and `.skip` to `describe` (`describe.only.bind(describe)`)? Or maybe it doesn't matter, it seems to be working.